oojs.js

```diff
@@ -10,3 +10,22 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+
+
+class Playlist {
+  constructor(title, songs) {
+  this.title = thisTitle;
```

Comment:
what does `thisTitle` represent? I think you just want `this.title = title` since that's was you called the parameter you passed in...